cdk/README.md

```diff
@@ -1,6 +1,11 @@
-# Welcome to your CDK TypeScript project!
+# Getting Started
+1. `npm install -g aws-cdk`
+2. `npm install -g typescript`
+3. `npm install @aws-cdk/core @aws-cdk/aws-lambda @aws-cdk/aws-lambda-event-sources @aws-cdk/aws-sqs`
```

Comment:
Do we need to update this every time there's a new CDK resource type, i.e. EC2, DynamoDB, Secrets Manager? Is there an NPM package that pulls in all of them? I'd rather do that than try to manage this in a bunch of places. 

Related question, can't these just be declared as devDependencies in the `package.json` file so that they can be used at runtime? Or maybe I just don't understand how that works. I want to avoid installing CDK and typescript globally. Ideally, these dependencies should be specific to the project.

Reply:
> Is there an NPM package that pulls in all of them?
So there is [monocdk](https://www.npmjs.com/package/monocdk) which holds a lot of these resources in one NPM package and we can move to using monocdk if that is preferrable. 

> can't these just be declared as devDependencies in the package.json file so that they can be used at runtime?  I want to avoid installing CDK and typescript globally.

So for this I was just following the steps from the [CDK prerequisites](https://docs.aws.amazon.com/cdk/latest/guide/work-with.html#work-with-prerequisites) and [the CDK with typescript guide](https://docs.aws.amazon.com/cdk/latest/guide/work-with-cdk-typescript.html). It may be possible to not install them globally and just have them as dev dependencies but I haven't seen that. I can look into this some more.

Reply:
1. Now that you mention it, my team uses monocdk and it was nice that I didn't have to worry about this - it's why I asked, actually. If we were only using like 2-3 services total in the CDK, then I could see doing this approach, but there's some stuff that is missing from here, so might as well move to monocdk. This is not a blocker to merge the PR, but I think the second comment is.
2. Would definitely prefer to explore options on this front since our global versions of CDK/Typescript will probably be different than someone contributing fresh to the repo a week or a month etc from now. If we can contain the dependency management within the repo, that'd be ideal. I thought about that for Gradle, since there are some deprecated functionalities in Gradle 6 I think that I'm using, but also have been ignoring that. Also, that's not entirely the same problem.